paymentsource/client_test.go

```diff
@@ -5,6 +5,8 @@ import (
 
 	stripe "github.com/stripe/stripe-go"
 	"github.com/stripe/stripe-go/card"
+	"github.com/stripe/stripe-go/bankaccount"
+	"github.com/stripe/stripe-go/token"
```

Comment:
Actually, would you mind trying to run the changes through `gofmt`? It'll automatically re-order some of the imports like this one.